challenge-1.js

```diff
@@ -0,0 +1,60 @@
+// BruteForce O(n^2)
+
+function primeNumbersBruteForce(n) {
+  if (typeof n != "number") return "bad request!";
+  let primeNumbers = [];
+  if (n >= 2) primeNumbers.push(2);
+  for (let index = 3; index <= n; index += 2) {
+    if (isPrime(index)) primeNumbers.push(index);
```

Comment:
Please checkout one line if using the `&&` operator. Using `if` is OK, just check this out to learn more!